zer0-os/zOS
# feat(autocomplete-members-component): expose sub-label to search result items

### What does this do?
- exposes `subLabel` to search result items for the `AutocompleteMembers` component.

### Why are we making this change?
- to display the users `primaryZID` (or, after some follow up changes, the users `publicWalletAddress`).

### How do I test this?
- on the create conversation panel/start group panel/ add members panel, search users that have a primaryZID set and check it is displayed as per screenshot below.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


Sub-Labels:
<img width="318" alt="Screenshot 2024-02-06 at 12 30 29" src="https://github.com/zer0-os/zOS/assets/39112648/4436304d-5b43-43b6-ae51-445a79eda8ef">
<img width="318" alt="Screenshot 2024-02-06 at 12 30 23" src="https://github.com/zer0-os/zOS/assets/39112648/225b3ff9-70d6-4635-a907-fe0769a45d99">
<img width="318" alt="Screenshot 2024-02-06 at 12 30 18" src="https://github.com/zer0-os/zOS/assets/39112648/7e5b5ac1-cdd3-4119-80b6-3e2d8f18f994">
